src/EnergyPlus/PluginManager.cc

```diff
@@ -454,7 +499,7 @@ namespace PluginManagement {
 #if LINK_WITH_PYTHON == 1
         for (auto &plugin : plugins) {
             if (plugin.runDuringWarmup || !DataGlobals::WarmupFlag) {
-                bool const didOneRun = plugin.run(iCalledFrom);
+                bool const didOneRun = plugin.run(state, iCalledFrom);
```

Comment:
Since we pass the current `state` to the plugin functions, we have to pass `state` around to a few new places in the code -- not many though.